dotnet/src/Experimental/Agents/ChatCompletion/ChatCompletionAgent.cs

```diff
@@ -0,0 +1,124 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.ChatCompletion;
+
+namespace Microsoft.SemanticKernel.Experimental.Agents;
+
+/// <summary>
+/// Represent an agent that is built around the SK ChatCompletion API and leverages the API's capabilities.
+/// </summary>
+public sealed class ChatCompletionAgent : KernelAgent
```

Comment:
No Description property?  I'd expect this to be part of the core KernelAgent...along with optional name?

Reply:
Not yet, we can always add them later when we know what they are for and how they will be used.

Reply:
+1 to including a description property; this will be necessary to make Agents as plugins feasible.